site-builder.js

```diff
@@ -110,14 +113,16 @@ function SiteBuilder(opts, buildLogger, doneCallback) {
 
 SiteBuilder.prototype.build = function() {
   var that = this;
-  fs.exists(this.sitePath, function(exists) {
-    (exists === true ? that.syncRepo() : that.cloneRepo())
-      .then(function() { return that.checkForFile('_config.yml'); })
-      .then(function(useJekyll) {
-        return (useJekyll === true ? that._buildJekyll() : that._rsync());
-      })
-      .then(that.done, that.done);
-  });
+  this.updateLock.doLockedOperation(function(done) {
+    fs.exists(that.sitePath, function(exists) {
+      (exists === true ? that.syncRepo() : that.cloneRepo())
```

Comment:
I suppose it could. I think it's part of my Ruby paranoia taking hold, where things like `0` and `''` are not considered falsy. Done.